top_films.py

```diff
@@ -0,0 +1,52 @@
+import connections as con
+from pyspark.sql import functions as f
+from datetime import datetime
```

Comment:
Imports should be grouped in the following order:
 
- Standard library imports.
- Related third party imports.
- Local application/library specific imports.

You should put a blank line between each group of imports.